docs/Using-Fleet/FAQ.md

```diff
@@ -1,6 +1,6 @@
 # Using Fleet FAQ
 
-- [What do I need to do to switch from Kolide Fleet to FleetDM Fleet?](#waht-do-i-need-to-do-to-switch-from-kolide-fleet-to-fleetdm-fleet)
+- [How can I switch to Fleet from Kolide Fleet?(#how-can-i-switch-to-fleet-from-kolide-fleet)
```

Comment:
```suggestion
- [How can I switch to Fleet from Kolide Fleet?](#how-can-i-switch-to-fleet-from-kolide-fleet)
```

Reply:
@Desmi-Dizney, the link was missing the closing square bracket. I have added it for you 👍🏻